src/artillery/order.tsx

```diff
@@ -0,0 +1,18 @@
+import { TRANSACTION_URI } from './API';
+import { postWithToken } from './tokens';
+import { CartItem } from '../types/StoreObject';
+
+type OrderLineFormat = {
```

Comment:
```suggestion
interface OrderLineFormat {
```

Seems like most code uses `interface` for object types, so should probably standardise and settle for one.

Reply:
I believe using a mixture of the two works, but I agree that it all should be reworked.
If they are simple types, I believe `type´ is sufficient. It is also easier to make function types.
If something requires extensions, I think that may be a better solution. Instead of making union types (as I have sometimes done), using interfaces to extend looks and feels more clean.

Reply:
As it is more of a visual cleanup I think it can be done at a different time. #10